output/index.html

```diff
@@ -24,21 +24,159 @@
                     <i class="fa-solid fa-bars"></i>
                 </nav>
             </header>
-            <div class="background">
-                <section class="text-container">
-                    <h1 class="title">I'm Linh Chi. Glad to see you!</h1>
+            <section class="background">
+                <div class="text-container">
+                    <h1 class="title">I'm Mariane. Glad to see you!</h1>
                     <p class="paragraph">I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
-                </section>
-                <section>
+                </div>
+                <div>
                     <div class="connect">LET'S CONNECT</div>
                     <ul class="social-media">
                         <li class="media-item"><i class="fa-brands fa-twitter"></i></li>
                         <li class="media-item"><i class="fa-brands fa-linkedin-in"></i></li>
                         <li class="media-item"><i class="fa-brands fa-github"></i></li>
                         <li class="media-item"><i class="fa-brands fa-facebook"></i></li>
                     </ul>
-                </section>
+                </div>
+            </section>
+            <div class="space">
+
             </div>
+            <div class="works">
+            <section class="cardwork">
+                <div class="snapshot">
+                </div>
+                <div class="info">
```

Comment:
- [ ] Good job creating works class :clap:, that contains projects in your portfolio, I would like to suggest you one thing , You should make sure that each card is centered and make sure that the margin `left and right` is the as as in the Figma design
Your design 
![Screenshot from 2023-01-13 16-53-06](https://user-images.githubusercontent.com/80458199/212349440-73ef57cd-8a89-4074-9b48-0f5a413da960.png)

Figma design 
![Screenshot from 2023-01-13 16-53-59](https://user-images.githubusercontent.com/80458199/212349662-ae4f8cc0-04c0-4547-a14a-1021941ecca8.png)